gui/default/syncthing/core/notifications.html

```diff
@@ -41,3 +41,31 @@ <h3 class="panel-title"><span class="fa fa-flash"></span>&nbsp;<span translate>A
     </div>
   </div>
 </notification>
+
+<notification id="fsWatcherNotification">
+  <div class="panel panel-success">
+    <div class="panel-heading">
+      <h3 class="panel-title"><span class="fa fa-flash"></span>&ensp;<span translate>Watching for Changes</span></h3>
+    </div>
+    <div class="panel-body">
+      <p translate>Contiuously watching for changes is now available within Syncthing. This will detect changes on disk and issue a scan on only the modified paths. The benefits are that changes are propagated quicker and that less full scans are required.</p>
+      <p><a href="https://docs.syncthing.net/users/syncing.html#scanning"><span class="fa fa-fw fa-book"></span>&nbsp;<span translate>Learn more</span></a></p>
+      <p>
+        <span translate>Do you want to enable watching for changes for all your folders?</span><br/>
+        <span translate>Additionally the full rescan interval will be increased (times 60, i.e. new default of 1h). You can also configure it manually for every folder later after choosing No.</span>
+      </p>
+      <p translate translate-value-syncthing-inotify="syncthing-inotify">Warning: If you are using an external watcher like {%syncthingInotify%}, you should make sure it is deactivated.</p>
+    </div>
+    <div class="panel-footer clearfix">
+      <div class="pull-right">
+        <button type="button" class="btn btn-primary btn-sm" ng-click="activateAllFsWatchers(); dismiss()">
```

Comment:
I suspect this might not work due to angular scoping rules.

Reply:
What do you suspect not to work? It does what I expect it to.

Reply:
You did test it, and it did work, right?

Reply:
Correct, as in the notification vanished and fs watcher was enabled and rescan interval adjusted.

Reply:
Right, ok.